corporate/lib/stripe.py

```diff
@@ -1305,6 +1324,21 @@ def process_initial_upgrade(
                 unit_amount=price_per_license,
             )
 
+            if customer.flat_discounted_months > 0:
+                num_months = 12 if billing_schedule == CustomerPlan.BILLING_SCHEDULE_ANNUAL else 1
+                flat_discounted_months = min(customer.flat_discounted_months, num_months)
+                discount = customer.flat_discount * flat_discounted_months
+                customer.flat_discounted_months -= flat_discounted_months
+                customer.save(update_fields=["flat_discounted_months"])
+
+                stripe.InvoiceItem.create(
+                    currency="usd",
+                    customer=customer.stripe_customer_id,
+                    description=f"Customer flat discount for {flat_discounted_months} months",
```

Comment:
This description probably should be clearer.

Reply:
Changed it to `description=f"Flat discount for {flat_discounted_months} months applied to the customer's subscription",`